src/rez/tests/test_suites.py

```diff
@@ -135,6 +139,31 @@ def test_3(self):
 
         self._test_serialization(s)
 
+    @per_available_shell()
+    @install_dependent()
+    def test_executable(self):
+        """Test tool can be executed"""
+        c_pooh = ResolvedContext(["pooh"])
+        s = Suite()
+        s.add_context("pooh", c_pooh)
+
+        expected_tools = set(["hunny"])
+        self.assertEqual(set(s.get_tools().keys()), expected_tools)
+
+        per_shell = config.get("default_shell")
+        suite_path = os.path.join(self.root, "test_suites", per_shell, "pooh")
+        s.save(suite_path)
+
+        bin_path = os.path.join(suite_path, "bin")
+        env = os.environ.copy()
+        # To access _rez_fwd
+        env["PATH"] = os.pathsep.join([system.rez_bin_path, env["PATH"]])
+        # activate suite
+        env["PATH"] = os.pathsep.join([bin_path, env["PATH"]])
+
+        output = subprocess.check_output(["hunny"], shell=True, env=env,
```

Comment:
Should have change this to use tool's `context.execute_shell(block=True)`, then I won't have to put `rez_bin_path` manually. Didn't know that before.

Will fix that later.

Reply:
Sorry, scratch that. The purpose of this test case is to test suite tool can be executed from a rez activated shell, not a resolved context.
Doc string added in f4ca471 to be more clear.